3.5/sdk/windowsservercore-1903/Dockerfile

```diff
@@ -57,6 +50,19 @@ RUN `
     `
     # Workaround VS installer/ngen issues
     && \Windows\Microsoft.NET\Framework\v4.0.30319\ngen uninstall "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\TestAgent\Common7\IDE\VSWebLauncher.exe" `
+    && \Windows\Microsoft.NET\Framework\v4.0.30319\ngen uninstall "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\Roslyn\csc.exe.config" `
```

Comment:
I'm confused by this. Can you elaborate on why it's useful?

My understanding is that since this isn't a managed assembly, it can't ever be ngened, so uninstalling it will always fail, short-circuiting the rest of this statement.

Reply:
It's a workaround for https://github.com/dotnet/roslyn/issues/38296